lib/logger.js

```diff
@@ -142,12 +143,18 @@ function Logger(key, options) {
 
     const useHttps = configs.AGENT_PROTOCOL === 'https';
 
+    const headers = (typeof window === 'object') ? {
+            ...configs.DEFAULT_REQUEST_HEADER
+            , source: `${options.UserAgent}/${pkg.version} (${browser.name}/${browser.version})`
+        } : {
+            ...configs.DEFAULT_REQUEST_HEADER
+            , 'user-agent': options.UserAgent || configs.DEFAULT_USER_AGENT
```

Comment:
When I tried using `detect-browser` in my terminal, the output:
```
> const browser = require('detect-browser').detect()
undefined
> browser
NodeInfo { version: '8.11.3', name: 'node', os: 'darwin' }
```

I think we can add this `node/8.11.3` info for non-browser cases as well - in other words, it's gonna be consistent for both `browser` and non-browser cases.